lib/miq_automation_engine/engine/miq_ae_method_service/miq_ae_service_model_base.rb

```diff
@@ -245,6 +246,13 @@ def initialize(obj)
       raise MiqAeException::ServiceNotFound, "#{ar_klass.name} Object [#{obj}] not found" if @object.nil?
     end
 
+    def show_url
+      default_url_options[:host] = MiqRegion.my_region.remote_ui_url
```

Comment:
It would be great if there was a configuration setting that could specific a load balancer address and fall back to this address if it was empty.  cc @gtanzillo, @Fryguy

Reply:
@gmcculloug 
We allow for overrides of the web services url by reading from the settings.yml
https://github.com/ManageIQ/manageiq/blob/e915a3f7ef305c4a3d12cdb06b37e1d39902f6b2/app/models/miq_region.rb#L247
Maybe we can add the logic to do the same for remote_ui_server